ddtrace/opentracer/tracer.py

```diff
@@ -265,7 +288,7 @@ def start_span(
         # create a new otspan and ddspan using the ddtracer and associate it
         # with the new otspan
         ddspan = self._dd_tracer.start_span(
-            name=operation_name,
+            name=operation_name,  # type: ignore[arg-type]
```

Comment:
why the ignore?

Reply:
This is actually a little strange, our opentracer `start_span()` method accepts an Optional operation name, while our ddtrace tracer requires a value. I'll create a ticket for resolving this issue, but this will (probably?) be a breaking change if we change the opentracer method to require a value for `operation_name`.

Reply:
yes it would be a breaking change